content/doc/tutorials/build-a-node-js-and-react-app-with-npm.adoc

```diff
@@ -140,89 +117,53 @@ process.
+the `npm` command to ensure that all dependencies required to run your application have been downloaded to the `node_modules` workspace directory.
+
+. Save your edited `Jenkinsfile` and commit it to your local `simple-node-js-react-npm-app` Git repository.
+E.g. Within the  `simple-node-js-react-npm-app` directory, run the commands: +
   `git add .` +
   then +
-  `git commit -m "Add initial Jenkinsfile"`
-. Go back to Jenkins again, log in again if necessary and click *Open Blue
-  Ocean* on the left to access Jenkins's Blue Ocean interface.
-. In the *This job has not been run* message box, click *Run*, then quickly
-  click the *OPEN* link which appears briefly at the lower-right to see Jenkins
-  building your Pipeline project. If you weren't able to click the *OPEN* link,
-  click the row on the main Blue Ocean interface to access this feature. +
-  *Note:* You may need to wait several minutes for this first run to complete.
-  After making a clone of your local `simple-node-js-react-npm-app` Git
-  repository itself, Jenkins:
-.. Initially queues the project to be run on the agent.
-.. Downloads the Node Docker image and runs it in a container on Docker.
-+
+  `git commit -m "Add initial Jenkinsfile"` +
+ and finally +
+`git push` to push your changes to your forked repository on GitHub, so it can be picked up by Jenkins.
+
+. Now click on *Build Now* on the left pane of your Pipeline project in Jenkins.
+After making a clone of your local `simple-node-js-react-npm-app` Git repository itself, Jenkins:
+
+. Initially queues the project to be run on the agent.
+. Runs the `Build` stage defined in the `Jenkinsfile` on the agent.
+
+During this time, npm downloads and installs many artifacts necessary to build your node/js application, which are ultimately stored in Jenkins' local `npm` repository.
 [.boxshadow]
-image:tutorials/node-js-react-01-downloading-node-docker-image.png[alt="Downloading
-Node Docker image",width=100%]
-.. Runs the `Build` stage (defined in the `Jenkinsfile`) on the Node container.
-   During this time, `npm` downloads many dependencies necessary to run your
-   Node.js and React application, which will ultimately be stored in the
-   `node_modules` workspace directory (within the Jenkins home directory).
-+
+image:tutorials/node/npm-install-logs.png[alt="npm install",width=100%]
+You can now click on *#1* to see the details of the build. You will then see how much time the build took waiting in the queue, and how much time it took to run.
 [.boxshadow]
-image:tutorials/node-js-react-02-initial-pipeline-downloading-npm-dependencies.png[alt="Downloading
-'npm' dependencies",width=100%]
+image:tutorials/node/build-details.png[alt="Build details",width=100%]
 
-+
-The Blue Ocean interface turns green if Jenkins built your Node.js and React
-application successfully.
+On the left, you can click on *Pipeline Overview* to see the stages of the Pipeline.
```

Comment:
```suggestion
In the left navigation pane, you can select *Pipeline Overview* to see the stages of the Pipeline.
```
Suggestion to provide more clear direction for users and using "select" over "click".